mlflow/store/tracking/rest_store.py

```diff
@@ -300,13 +305,41 @@ def get_experiment_by_name(self, experiment_name):
             return None
 
     def log_batch(self, run_id, metrics, params, tags):
```

Comment:
Rather than modifying `RestStore`, can we move these changes to `MlflowClient.log_batch()`?

Reply:
In my understanding, we cannot detect the case, where [`_validate_param_keys_unique`](https://github.com/mlflow/mlflow/blob/17797ec97d9b4be355a756bb99efb7117c1b71a4/mlflow/utils/validation.py#L211-L226) raises `MlflowException` due to duplicated key params, by chunking batch data before calling `_validate_param_keys_unique` from `log_batch` on `mlflow/store/tracking/file_store.py` and `mlflow/store/tracking/sqlalchemy_store.py`. This was why I modified `RestStore` rather than `MLflowClienet.log_batch`.

Reply:
Thank you for your summary. I've moved the change in `rest_store.py` to the `MlflowClient` part.